### PR TITLE
V1/5 dockerfix

### PR DESCRIPTION
**Pull request by SundayBraid**
- add nodeJs and composer inside laravel container
- hardcode UID and GID directly to dockerfile to avoid using argument

### CODE REVIEW

File: infra/dev/laravel/Dockerfile [hunk 1]
```diff
@@ -3,17 +3,35 @@ FROM php:7.4-fpm
...
-ARG USER_ID
-ARG GROUP_ID
+# Raw code till i haven't found a way to pass .env file values
+ENV USER_ID 1000
+ENV GROUP_ID 1000
 
 RUN groupadd -f $GROUP_ID
 RUN useradd -u $USER_ID -g $GROUP_ID user
 
+RUN mkdir /home/user
```
Comment (kayoumido) [inline infra/dev/laravel/Dockerfile:13]: Why do we need to create a home directory?

File: infra/dev/laravel/Dockerfile [hunk 2]
```diff
@@ -3,17 +3,35 @@ FROM php:7.4-fpm
...
+
+RUN chown -R $USER_ID:$GROUP_ID /home/user
```
Comment (kayoumido) [inline infra/dev/laravel/Dockerfile:15]: Same question as line 13

File: infra/dev/laravel/Dockerfile [hunk 3]
```diff
@@ -3,17 +3,35 @@ FROM php:7.4-fpm
...
+
+RUN chown -R $USER_ID:$GROUP_ID /var/www
```
Comment (kayoumido) [inline infra/dev/laravel/Dockerfile:17]: same

File: infra/dev/docker-compose.yml [hunk 4]
```diff
@@ -12,40 +12,18 @@ services:
...
-        GROUP_ID: ${GROUP_ID}
     container_name: dev_laravel
     volumes:
       - ../../src:/var/www
       - ./laravel/php-fpm.conf:/usr/local/etc/php-fpm.d/zz-docker.conf
     ports:
       - 42090:9000
     networks:
-      - paul_net
+      - paul_net
```
Comment (kayoumido) [inline infra/dev/docker-compose.yml]: :triggered:


:D

File: infra/dev/laravel/Dockerfile (hunk 3 above)
Comment (SundayBraid) [reply infra/dev/laravel/Dockerfile:17]: In case it hasn't the good right.

File: infra/dev/laravel/Dockerfile (hunk 2 above)
Comment (SundayBraid) [reply infra/dev/laravel/Dockerfile:15]: 'cause user doesn't have the right over this folder

File: infra/dev/laravel/Dockerfile (hunk 1 above)
Comment (SundayBraid) [reply infra/dev/laravel/Dockerfile:13]: Because it doesn't exist, needed by npm (error if not exist)

File: infra/dev/docker-compose.yml (hunk 4 above)
Comment (kayoumido) [reply infra/dev/docker-compose.yml]: I'll remove it my self..otherwise I won't be able to sleep D:

**Review by kayoumido — APPROVED** [review]
All gucci

**Comment by kayoumido** [comment]
We'll have to see if we can move `UID` and `GID` into an `.env` but it can be done later on

**Pull request merged**